.github/ISSUE_TEMPLATE/umbrella.yml

```diff
@@ -0,0 +1,80 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Umbrella
+title: "[Umbrella] "
+description: An umbrella issue with multiple sub-tasks
+labels: [ "umbrella" ]
+body:
+
+  - type: checkboxes
+    attributes:
+      label: Code of Conduct
+      description: The Code of Conduct helps create a safe space for everyone. We require that everyone agrees to it.
+      options:
+        - label: >
+            I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: Search before asking
+      options:
+        - label: >
+            I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues) and found no similar
+            issues.
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Describe the proposal
+      description: >
+        Kindly also put the related link here if it is a [KPIP](https://kyuubi.apache.org/improvement-proposals.html).
+      placeholder: >
+        Please describe the content of the proposal clearly and concisely.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Task list
+      description: >
+        For more details, kindly refer to [github docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists).
```

Comment:
`kindly` to `please`

Reply:
They are synonyms actually. Updated anyway.